client/my-sites/subscribers/main.tsx

```diff
@@ -1,22 +1,58 @@
+import { Button, Gridicon } from '@automattic/components';
 import { useLocalizeUrl } from '@automattic/i18n-utils';
 import { translate } from 'i18n-calypso';
 import page from 'page';
 import { useSelector } from 'react-redux';
+import { navItems } from 'calypso/blocks/stats-navigation/constants';
 import { Item } from 'calypso/components/breadcrumb';
 import DocumentHead from 'calypso/components/data/document-head';
 import Main from 'calypso/components/main';
+import NavigationHeader from 'calypso/components/navigation-header';
 import { SubscriberListContainer } from 'calypso/my-sites/subscribers/components/subscriber-list-container';
-import { SubscribersPageProvider } from 'calypso/my-sites/subscribers/components/subscribers-page/subscribers-page-context';
+import {
+	SubscribersPageProvider,
+	useSubscribersPage,
+} from 'calypso/my-sites/subscribers/components/subscribers-page/subscribers-page-context';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
 import { AddSubscribersModal } from './components/add-subscribers-modal';
-import { SubscribersHeader } from './components/subscribers-header';
+import { SubscribersHeaderPopover } from './components/subscribers-header-popover';
 import { UnsubscribeModal } from './components/unsubscribe-modal';
 import { SubscribersFilterBy, SubscribersSortBy } from './constants';
 import { getSubscriberDetailsUrl, getSubscribersUrl } from './helpers';
 import { useUnsubscribeModal } from './hooks';
 import { Subscriber } from './types';
+
 import './style.scss';
 
+type SubscribersHeaderProps = {
+	selectedSiteId: number | undefined;
+	navigationItems: Item[];
+};
+
+const SubscribersHeader = ( { navigationItems, selectedSiteId }: SubscribersHeaderProps ) => {
+	const { setShowAddSubscribersModal } = useSubscribersPage();
+
+	return (
+		<NavigationHeader
+			className="stats__section-header modernized-header"
+			title={ translate( 'Jetpack Stats' ) }
+			subtitle={ translate( "View your site's performance and learn from trends." ) }
```

Comment:
This one doesn't look quite right. Check out https://wordpress.com/subscribers/:siteSlug for comparison.

Reply:
It's fixed now. Thanks for noting it